docs/specs/git.yml

```diff
@@ -62,13 +62,19 @@ outputs:
 # Specify invalid author in YAML header should log warning
 noDryRun: true
 usePhysicalInput: true
-inputs:
-  docs/index.md: |
-    ---
-    author: invalid_user
-    ---
-    # git-test
-  docfx.yml:
+repos:
+  https://github.com/testowner/testrepo-with-invalid-user#main:
```

Comment:
why we need to update this test case?

Reply:
Because I cannot find an easy to prevent it to finally find the Docfx repo itself when dinIt's because we want to scan all redirection files within the repo folder, the _buildOptions.repopath is from [here (link)](https://github.com/dotnet/docfx/blob/6163e1c90c8f316789051d511bb1f8a9636ecae5/src/docfx/lib/git/GitUtility.cs#L17), and for this test, I cannot find an easy to prevent it from finally finding the Docfx repo itself while let it use the cache.